package.json

```
@@ -9,16 +9,21 @@
     "watch-hacks": "npm-watch build-hacks",
     "watch-html": "npm-watch build-html",
     "build": "npm run build-hacks&&npm run build-html",
-    "start": "concurrently 'npm:watch-*'"
+    "start": "concurrently 'npm:watch-*'",
+    "update-bitsy": "node fetch-bitsy-source.js",
+    "postinstall": "node fetch-bitsy-source.js"
```

better to call the other package script directly when doing stuff like this; reduces the likelihood one gets updated and the other is forgotten
```suggestion
    "postinstall": "npm run update-bitsy"
```